testutil/sims/state_helpers.go

```diff
@@ -257,7 +259,11 @@ func AppStateFromGenesisFileFn(r io.Reader, cdc codec.JSONCodec, genesisFile str
 
 	var genesis genutiltypes.AppGenesis
 	if err = json.Unmarshal(bytes, &genesis); err != nil {
-		return genesis, nil, err
+		// NOTE: cometbft uses a custom JSON decoder for GenesisDoc
+		// On some machines, the above will go wrong, so try using cbftjson for Unmarshal again.
+		if err = cbftjson.Unmarshal(bytes, &genesis); err != nil {
```

Comment:
We have an helper for that un x/gentutil

Reply:
Thanks for the suggestion, have used the AppGenesisFromReader function to Unmarshal the genesis.json file

Reply:
Yes, there is some truth to what you said. But as I mentioned above: "**Add the Sigverify flag in the simulation test whether to check the transaction signature.**", in order to achieve this requirement, I followed the previous implementation and added a **FlagSigverify** global variable（x/simulation/client/cli/flags.go）, and during in the simulation the test, I need to verify the user's signature based on this global variable. In order to achieve this requirement, I have not thought of a better implementation for the time being. Can you give me any more suggestions or tips?